server/views/pages/recategoriserHome.html

```diff
@@ -42,81 +45,196 @@ <h1 class="govuk-heading-l">{{ pageTitle }}</h1>
+            </div>
+            <div
+              id="selectAllSuitabilityForOpenConditionsFilter"
+              class="linkStyledButton govuk-!-font-size-16 govuk-!-margin-bottom-3"
+            >
+              {{ 'Select all' if filters.suitabilityForOpenConditions|length < allFilters.suitabilityForOpenConditions|length else 'Deselect all' }}
+            </div>
+            {{ govukCheckboxes({
+              name: 'suitabilityForOpenConditions[]',
+              classes: "govuk-checkboxes--small",
+              fieldset: {
+                legend: {
+                  text: "Filter based on suitability for open conditions",
+                  classes: "govuk-visually-hidden"
+                }
+              },
+              items: filterCheckboxes
+            }) }}
+
+            <div class="replacementApplyFilterButtonContainer">
+              {{ govukButton({
+                id: 'applyFilters',
+                type: 'submit',
+                text: "Apply filters",
+                classes: "govuk-!-margin-top-3"
+              }) }}
+            </div>
+          {% endset -%}
+
+          {% set filterCategories = [] %}
+          {% if filters.suitabilityForOpenConditions.length > 0 %}
+            {% set selectedItems = [] %}
+            {% for filter in filters.suitabilityForOpenConditions %}
+              {% set startPositionOfFilterInUrl = fullUrl.indexOf(filter) - (28 + 7) %}
+              {% set removalHref = fullUrl.substring(0, (startPositionOfFilterInUrl - 1) if fullUrl[startPositionOfFilterInUrl - 1] == '&' or filters.suitabilityForOpenConditions.length == 1 else startPositionOfFilterInUrl) + fullUrl.substr(fullUrl.indexOf(filter) + filter.length + (0 if fullUrl[startPositionOfFilterInUrl - 1] == '&' else 1), fullUrl.length) %}
+              {% set selectedItems = (selectedItems.push({
+                href: removalHref,
+                text: allFilters['suitabilityForOpenConditions'][filter]
+              }), selectedItems) %}
+            {% endfor %}
+
+            {% set filterCategories = (filterCategories.push({
+              heading: {
+                text: 'Suitability for open conditions'
+              },
+              items: selectedItems
+          }), filterCategories) %}
+          {% endif %}
+
+          {{ mojFilter({
+            heading: {
+              text: 'Filter'
+            },
+            selectedFilters: {
+              heading: {
+                text: 'Selected filters'
+              },
+              clearLink: {
+                text: 'Clear filters',
+                href: '/recategoriserHome'
+              },
+              categories: filterCategories
+            } if filterCategories.length > 0,
+            optionsHtml: filterOptionsHtml,
+            submit: {
+              attributes: {
+                "data-test-id": "submit-button",
+                style: "display: none;"
```

Comment:
<img width="386" alt="Screenshot 2024-09-02 at 10 17 07" src="https://github.com/user-attachments/assets/6e4c0639-6a79-40be-b7ce-d18fa61cb8da">

i tried it without line 135 and it seems hidden by default anyway, so maybe you can just remove this line. If not, it's probably easiest to give this element an id, and then add a style to hide it from the stylesheet, which is valid for the csp